### PR TITLE
Updating some files

### PR DESCRIPTION
**Pull request by ItsLuiggiYahoo**
The last pull request was closed after noticing something wrong, i've changed it and it's now fine

Added Functions(methods) and some of the things that were missing.

> [!IMPORTANT]
> Some of the information i put had blank pages, i need somebody to put the information for me.

As aways feel free to change anything, i don't mind, be my guest!

### CODE REVIEW

**Comment by indexxing** [comment]
I will merge this in a few minutes after I make some small changes.

File: docs/index.html [hunk 1]
```diff
@@ -309,4 +309,4 @@ <h2 class="fw-bold start-journey-text" style="text-align: left;">Start your jour
       </div>
     </div>
   </body>
-</html>
```
Comment (indexxing) [inline docs/index.html:312]: what happened here lol

Comment (ItsLuiggiYahoo) [reply docs/index.html:312]: i don't remember removing the </html> thing...
i'll investigate

**Comment by indexxing** [comment]
Some notes:
- If you are ever adding an event, method, or property please put the description as TO-DO or something along those lines to make it more clear
- Make sure to capitalize the class names of what events, methods and properties return properly so they automatically get replaced with a link to the page of that return

Thanks for the pull request, I'll merge it in a minute.

**Pull request merged**